README.md

```diff
@@ -85,3 +85,53 @@ Which config to pick is determined through env variables:
 ### Logging
 
 Logging is set up (by default) through `config/logging-json.conf`.
+
+## Running tests
+
+You can run the unit tests with pytest, the usual way.
+
+    pytest
+
+This will only pick up the unit tests, which you can find in the directory `tests`.
+
+That is to say, when you run the command `pytest` without specifying a specific directory or file
+it will _not_ run the integration tests, only the unit tests.
+That is a deliberate choice because the integration tests could take long and they run
+against a backend server. We want to avoid that you have to wait a long time for each test run during your development cycle.
+
+So if you want integration tests then you have to run those separately, as described below.
+### Running integration tests
+
+To make it easier to run the integration test suite against the _default_ backend you can run the following shell script (from the root of your local git repository):
+
+    ./scripts/run-integration-tests.sh
+
+To run the integration test suite against any other OpenEO backend:
+
+- first, specify the backend base URL in environment variable `ENDPOINT` ,
```

Comment:
I know this is based on geopyspark integration tests, but I'd prefer to start fresh here with a more descriptive env var like `OPENEO_BACKEND_URL`

Reply:
Renamed the environment variable and using the better default URL.